spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -112,15 +112,21 @@ def add_group_to_build(app_id: nil, group_id: nil, build_id: nil)
     ##
 
     def post_tester(app_id: nil, tester: nil)
-      assert_required_params(__method__, binding)
+      return create_app_level_tester(app_id: app_id,
```

Comment:
I think this could live on `Spaceship::TestFlight::Tester` rather than adding this method to the client, that way the client is really just an interface into the iTC API rather than having semantics different from the request specs.